examples/st_game/memory/retrive.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# @Desc   : Retrive函数实现
+
+from numpy import dot
+from numpy.linalg import norm
+from datetime import datetime
+from associative_memory import AgentMemory,MemoryBasic
+from utils.utils import embedding_tools
+
+def agent_retrive(agent:AgentMemory,currtime:datetime,memory_forget:float,query:str,n:int= 30,topk:int=4) -> list[MemoryBasic]:
```

Comment:
`agent:AgentMemory` agent is agent, memory is memory, not the same. And you can do the retrieve inside `XXXMemory` class. Use `retrieve` not `retrive`.